app/controllers/events_controller.rb

```diff
@@ -36,6 +36,14 @@ def update
     end
   end
 
+  def feed
+    @events = Event.order(date: :desc).limit(20)
+
+    respond_to do |format|
+      format.xml
+    end
+  end
```

Comment:
Hey @enderahmetyurt, thanks for working on this!

We have been trying to scope the controllers and only use the 7 default actions.

What do you think about moving this to a `Events::FeedController` and have the current logic on the `index` action?

Reply:
Hello @marcoroth Thank you for your feedback. I see, I will do it.